bitnami/contour/values.yaml

```diff
@@ -1315,6 +1313,17 @@ metrics:
     selector: {}
     labels: {}
 
+  ## Prometheus Operator prometheusRules
+  ## @param metrics.prometheusRule.enabled Creates a Prometheus Operator prometheusRule (also requires `metrics..serviceMonitor.enabled` to be `true` and `metrics.prometheusRule.rules`)
+  ## @param metrics.prometheusRule.namespace Namespace for the prometheusRule Resource (defaults to the Release Namespace)
+  ## @param metrics.prometheusRule.additionalLabels Additional labels that can be used so prometheusRule will be discovered by Prometheus
+  ## @param metrics.prometheusRule.rules Prometheus Rule definitions
+  prometheusRule:
```

Comment:
Could you follow the same format we use in other charts (e.g. Apache) for this kind of object? See
https://github.com/bitnami/charts/blob/main/bitnami/apache/values.yaml#L686